addons/website_event_questions/tests/test_event_internals.py

```diff
@@ -29,3 +31,32 @@ def test_event_type_configuration(self):
         self.assertEqual(
             set(event.general_question_ids.mapped('answer_ids.name')),
             set(['Q2-Answer1', 'Q2-Answer2']))
+
+    def test_process_attendees_form(self):
+        event = self.env['event.event'].create({
+            'name': 'Event Update Type',
+            'event_type_id': self.event_type_complex.with_user(self.env.user).id,
+            'date_begin': FieldsDatetime.to_string(datetime.today() + timedelta(days=1)),
+            'date_end': FieldsDatetime.to_string(datetime.today() + timedelta(days=15)),
+        })
+
+        form_details = {
+            '1-name': 'Pixis',
+            '1-email': 'pixis@gmail.com',
+            '1-phone': '+32444444444',
+            '1-event_ticket_id': '2',
+            '2-name': 'Geluchat',
+            '2-email': 'geluchat@gmail.com',
+            '2-phone': '+32777777777',
+            '2-event_ticket_id': '3',
+            '0-answer_ids-3': '7',
```

Comment:
Maybe add answers specific to some registrations to check. Normally it is supported by frontend (continuing previous comment).

Reply:
@tde-banana-odoo Fixed, and I added a test, I forgot the option "ask each attendees" on the event question